lib/menu_loader.rb

```diff
@@ -0,0 +1,16 @@
+require 'dish'
+class MenuLoader
+  def initialize
+    @objects = []
+  end
+  def load(file)
+    fail "Data File missing!" if !File.exists? file
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)